content/apps/maintaining-github-apps/modifying-a-github-app-registration.md

```diff
@@ -79,7 +79,7 @@ If you select **Request user authorization (OAuth) during installation**, you wi
 
 You can change the access permissions that are granted to your {% data variables.product.prodname_github_app %} using the following steps.
 
-When you change the **repository** or **organization** permissions of an app, each account where the app is installed will need to approve the new permissions. When you change the **account** permissions of an app, each user that has authorized the app will need to approve the permission changes. Updated permissions won't take effect on an installation or user authorization until the new permissions are approved. You can use the [installation webhook](/webhooks-and-events/webhooks/webhook-events-and-payloads?actionType=new_permissions_accepted#installation) to find out when people accept new permissions for your app.
+When you change the **repository** or **organization** permissions of an app, each account where the app is installed will need to approve the new permissions. When you change the **account** permissions of an app, each user that has authorized the app will need to approve the permission changes. In both cases, the existing users of your app will receive the emails from GitHub which notify the change of the permissions and ask for reviewing those changes. Updated permissions won't take effect on an installation or user authorization until the new permissions are approved. You can use the [installation webhook](/webhooks-and-events/webhooks/webhook-events-and-payloads?actionType=new_permissions_accepted#installation) to find out when people accept new permissions for your app.
```

Comment:
Small wording change to use active voice instead of passive voice, use the [variable](https://docs.github.com/en/contributing/writing-for-github-docs/style-guide#reusables-and-variables) for "[GitHub](https://github.com/github/docs/blob/main/data/variables/product.yml#L13)", and to mention that the account receiving the email could be either a user account or an organization owner.

```suggestion
When you change the **repository** or **organization** permissions of an app, each account where the app is installed will need to approve the new permissions. When you change the **account** permissions of an app, each user that has authorized the app will need to approve the permission changes. In both cases, {% data variables.product.prodname_dotcom %} will send an email to each organization owner or user, notifying them of the request to update the app's permissions. Updated permissions won't take effect on an installation or user authorization until the new permissions are approved. You can use the [installation webhook](/webhooks-and-events/webhooks/webhook-events-and-payloads?actionType=new_permissions_accepted#installation) to find out when people accept new permissions for your app.
```